frontend/src/scenes/notebooks/Notebook/NotebookShare.tsx

```diff
@@ -0,0 +1,72 @@
+import { LemonBanner, LemonButton, LemonDivider } from '@posthog/lemon-ui'
+import { combineUrl } from 'kea-router'
+import { IconCopy } from 'lib/lemon-ui/icons'
+import { LemonDialog } from 'lib/lemon-ui/LemonDialog'
+import { copyToClipboard } from 'lib/utils'
+import posthog from 'posthog-js'
+import { useState } from 'react'
+import { urls } from 'scenes/urls'
+
+export type NotebookShareProps = {
+    shortId: string
+}
+export function NotebookShare({ shortId }: NotebookShareProps): JSX.Element {
+    const url = combineUrl(`${window.location.origin}${urls.notebook(shortId)}`).url
+
+    const [interestTracked, setInterestTracked] = useState(false)
+
+    const trackInterest = (): void => {
+        posthog.capture('pressed interested in notebook sharing', { url })
+    }
+
+    return (
+        <div className="space-y-2">
+            <h3>Internal Link</h3>
+            <p>
+                <b>Click the button below</b> to copy a direct link to this Notebook. Make sure the person you share it
+                with has access to this PostHog project.
```

Comment:
```suggestion
                <b>Click the button below</b> to copy a direct link to this Notebook. Make sure the person you share it with has access to this PostHog project.
```

Reply:
Ah you screwed me! That was prettier :D I had to revert the suggestion